compute/xmpp_wikibot/wikibot.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Wikibot server example using SleekXMPP client library"""
+
+import argparse
+import getpass
+import json
+import logging
+import threading
+import urllib
+
+from flask import Flask, request
+import requests
+import sleekxmpp
+
+
+app = Flask(__name__)
+
+@app.route('/send_message', methods=['GET'])
+def send_message():
+    recipient = request.args.get('recipient')
+    message = request.args.get('message')
+
+    if chat_client and recipient and message:
+        chat_client.send_message(mto=recipient, mbody=message)
+        return 'message sent to {} with body: {}'.format(recipient, message)
+    else:
+        logging.info('chat client or recipient or message does not exist!')
+        return 'message failed to send', 400
+
+
+def run_server(host='0.0.0.0'):
+    app.run(threaded=False, use_reloader=False, host=host)
```

Comment:
debug=False here, please. We shouldn't be using Flask's dev server to serve public traffic *at all*, but at least disable the debug flag.